tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -374,8 +374,8 @@ void ChangeNumberSequential(NumberSequential* p)
 
 bool IsCorrectNumberSequential(NumberSequential* p)
 {
-	if(p->i32 != (-0x7fffffff - 1) || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 || 
-		p->sb != 0x7f ||p->i16 != -0x8000 || p->ui16 != 0xffff || p->i64 != -1234567890 ||
+	if(p->i32 != (-0x7fffffff - 1) || p->ui32 != 0xffffffff || p->s1 != -0x8000 || p->us1 != 0xffff || p->b != 0 ||
+		p->sb != 0x7f ||p->i16 != -0x8000 || p->ui16 != 0xffff || p->i64 != (unsigned)-1234567890 ||
```

Comment:
Please use `uint32_t` instead of a naked `unsigned` - throughout.

Reply:
Changed to `uint32_t`.

Reply:
I had to change it to `uint64_t` (to match LHS: `i64` -> `LONG64` -> `unsigned long long`); good thing is that reverse P/Invoke caught the width issue: https://ci.dot.net/job/dotnet_coreclr/job/master/job/x64_checked_centos7.1_innerloop_tst_prtest/10911/consoleFull#-109078752979494335-f7bd-47d0-8771-8661e00c2db2 8-)